asset/css/mixin/mixins.less

```diff
@@ -0,0 +1,25 @@
+/*! Icinga Ipl Web | (c) 2021 Icinga GmbH | GPLv2 */
```

Comment:
Please remove this, it's for one not required and it's not licensed under the GPL but MIT.